app/source/css/views/Listing.styl

```diff
@@ -0,0 +1,119 @@
+/**
+Listing
+**/
+
+.Listing {
+    > * ul {
```

Comment:
@joshdanielson I'm not sure if you really needs to use the '> * ul' selector, aside from increasing specificity. I will left this but please review [this example at jsfiddle about using that selector and similar rules](https://jsfiddle.net/IagoSRL/41xv7su7/2/) to clearly see how it relates.